org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "command": "qelectrotech",
+  "rename-desktop-file": "qelectrotech.desktop",
+  "rename-appdata-file": "qelectrotech.appdata.xml",
+  "rename-icon": "qelectrotech",
+  "copy-icon": true,
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
```

Comment:
AFAIK the app works well in Wayland, so maybe change `--socket=x11` into `--socket=fallback-x11`

Reply:
yes, you can

Reply:
https://github.com/qelectrotech/qelectrotech-source-mirror/commit/153aecbbf311500d5eb0d340f3b9a4d58528562e

Reply:
Hi,
Do I need to edit the PR I submit or scorpio810 able to edit my PR?